### ISSUE TITLE
PCB schematic does not match pin description

### ISSUE THREAD

**Issue by bojanpotocnik** [bigtreetech/BIGTREETECH-SKR-mini-E3#71]
[BTT SKR mini E3 V1.2sch.pdf](https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/blob/master/hardware/BTT%20SKR%20MINI%20E3%20V1.2/BTT%20SKR%20mini%20E3%20V1.2sch.pdf) does not match [BTT SKR MINI E3 V1.2PIN.pdf](https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/blob/master/hardware/BTT%20SKR%20MINI%20E3%20V1.2/BTT%20SKR%20MINI%20E3%20V1.2PIN.pdf).

In the schematic SERVOS connector (J5) is marked as:
1. +5V
2. GND
3. SERVOS (PA1)

but on the pin description it is:
1. PA1 (SERVOS)
2. +5V
3. GND

This is clearly not an Altium part library mistake, as the same library part and schematic symbol is also used for PT-DET which matches the schematic.

This could also apply to the #64, however there is no defined pin 1 on the screw terminals so I cannot be 100 % sure what is the root cause of that error.

**Comment by brew99** [comment]
Yup, looks like they didn't update the sch when they went to the v1.2 board.   All should use the pin diagram
